integration_test/lib/main.dart

```diff
@@ -433,25 +433,30 @@ class _MyAppState extends State<MyApp> {
   }
 
   Future<void> testGetApptimizeTestInfo() async {
-    var result = await Apptimize.apptimizeTestInfo;
-
+    var result = await (Apptimize.apptimizeTestInfo
+        as FutureOr<Map<String, ApptimizeTestInfo?>?>);
+    assertNotNull(result);
+    if (result == null) return;
```

Comment:
do we need both?

Reply:
Yes - the static analyzer will complain otherwise (the throw in the assert doesn't appear to be enough for this to realize that the value will not be null).